command_server/command_server.py

```diff
@@ -0,0 +1,80 @@
+import asyncio
+import aiohttp
```

Comment:
just saw those imports and though - actually Aiohttp package is not default for Python 3, It seems reasonable to add some kind of requirements file.